python/running.md

```diff
@@ -6,51 +6,51 @@ a newer version along with various useful packages
 (see details on the [LCG stacks][lcg_stack]) by setting up an LCG environment:
 
 ```bash
-$ source /cvmfs/sft.cern.ch/lcg/views/setupViews.sh LCG_93 x86_64-slc6-gcc62-opt
+$ source /cvmfs/sft.cern.ch/lcg/views/setupViews.sh LCG_93python3 x86_64-slc6-gcc62-opt
```

Comment:
LCG_93 has the annoying property of not having a `python` binary, only `python3`.

Maybe we should use `LCG_94python3` instead here?